oauthproxy.go

```diff
@@ -351,7 +368,7 @@ func buildSessionChain(opts *options.Options, sessionStore sessionsapi.SessionSt
 
 	if opts.SkipJwtBearerTokens {
 		sessionLoaders := []middlewareapi.TokenToSessionFunc{
-			opts.GetProvider().CreateSessionFromToken,
+			opts.GetProvider(0).CreateSessionFromToken,
```

Comment:
I think we will want to add all providers to this rather than just the first one